sysutils/rdfind/Portfile

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                rdfind
-version             1.3.5
-revision            1
+github.setup        pauldreik rdfind 1.4.1 releases/
```

Comment:
That's not an intended use of the setup command. That last argument is not to change the directory, but instead only the filename. The normal way to get a release version is to add the line:

`github.tarball_from releases
`

Reply:
I am aware of the setup command's usage.  That was done intentionally as `releases/` is actually the version prefix being used for git tags within the [rdfind repository](https://github.com/pauldreik/rdfind):

![rdfind git tags](https://imgur.com/download/DuSFXbh)

When removed and `github.tarball_from releases` is added, MacPorts fails to find the distfile.

Reply:
OK, I guess this project does it uniquely!

Reply:
That it does 🤷🏾‍♂️🙂. Thanks @mf2k.